src/leaflet/core/leaflet.directive.ts

```diff
@@ -80,12 +80,10 @@ export class LeafletDirective
 	}
 
 	ngOnInit() {
-
 		// Create the map outside of angular so the various map events don't trigger change detection
+		this.map = map(this.element.nativeElement, this.options);
```

Comment:
Why was moving this line outside `this.zone.runOutsideAngular` needed?
It looks to me like Leaflet adds it's own listeners during Map creation/initialize: https://github.com/Leaflet/Leaflet/blob/master/src/map/Map.js#L142
If this is the case, then removing this from `runOutsideAngular` will affect the performance

Reply:
@danmana correct. This needs to be inside the runOutsideAngular block otherwise Angular will fire change detection on every map interaction event (e.g., mouse move, click, etc.)